_content/tour/ukr/slices.article

```diff
@@ -1,29 +1,29 @@
-Slices
-Slices are an incredibly important data structure in Go. They form the basis for how we manage and manipulate data in a flexible, performant, and dynamic way. It's very beneficial for all Go programmers to learn how slices work, and how to use them.
+Зрізи (Slices)
+Зрізи є надзвичайно важливою структурою даних у Go. Вони є основною того, як ми керуємо і маніпулюємо даними у гнучкий, продуктивний і водночас динамічний спосіб. Всім Go розробникам напрочуд важливо знати, як працюють зрізи і як їх використовувати.
 
-* Slices
+* Зрізи
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Дивитись відео]]
+- Якщо потрібна фінансова допомога, то використовуйте [[https://www.ardanlabs.com/scholarship/][формy для отримання стипендії]]
 
-Slices are an incredibly important data structure in Go. They form the basis for
-how we manage and manipulate data in a flexible, performant, and dynamic way. It's
-very beneficial for all Go programmers to learn how slices work, and how to use them.
+Зрізи є надзвичайно важливою структурою даних у Go. Вони є основною того,
+як ми керуємо і маніпулюємо даними у гнучкий, продуктивний і водночас динамічний спосіб. Всім
+Go розробникам напрочуд важливо знати, як працюють зрізи і як їх використовувати.
 
 .image /tour/eng/static/img/sl1.png
 
-** Code Review 
+** Перегляд коду
 
-- *Example* *1:* Declare and Length
-- *Example* *2:* Reference Types
-- *Example* *3:* Appending slices
-- *Example* *4:* Taking slices of slices
-- *Example* *5:* Slices and References
-- *Example* *6:* Strings and slices
-- *Example* *7:* Variadic functions
-- *Example* *8:* Range mechanics
-- *Example* *9:* Efficient Traversals
-- *Example* *10:* Three index slicing
+- *Приклад* *1:* Оголошення і Довжина
+- *Приклад* *2:* Типи посилань
+- *Приклад* *3:* Доповнення зрізів
+- *Приклад* *4:* Зрізаємо зрізи
+- *Приклад* *5:* Зрізи і посилання
+- *Приклад* *6:* Рядки і зрізи
+- *Приклад* *7:* Варіативні функції
+- *Приклад* *8:* Механіка меж
+- *Приклад* *9:* Ефективні обходи
+- *Приклад* *10:* Трьох-індексні зрізи
```

Comment:
https://slovnyk.ua/index.php?swrd=%D1%82%D1%80%D1%8C%D0%BE%D1%85%D1%96%D0%BD%D0%B4%D0%B5%D0%BA%D1%81%D0%BD%D0%B8%D0%B9

```suggestion
- *Приклад* *10:* Трьохіндексні зрізи
```